rattr/analyser/util.py

```diff
@@ -662,6 +663,31 @@ def __exit__(self, *_):
         pass
 
 
+class changes:
```

Comment:
I would still make this a `Changes` class even if it's to be used as a context-manager